crates/wit-component/tests/interfaces/diamond-disambiguate/shared1.wit.print

```diff
@@ -0,0 +1,5 @@
+interface shared {
```

Comment:
Do you think a future improvement could recover from the encoding the fact that this interface is `default` for this document?

Reply:
I'd like to, yeah, but in talking with Luke we haven't figure out a great way to encode it in wasm just yet.